update-samples.sh

```diff
@@ -0,0 +1,8 @@
+#Update sample apps to the latest
```

Comment:
This will be obsolete after the samples are removed and a new sample created.

Reply:
what should be done here?